### ISSUE TITLE
[Component amendment]: <CTA Block> Add CSS pseudo element for the single right-pointing angle quotation mark (›)

### ISSUE THREAD

**Issue by Sophie-32** [canonical/vanilla-framework#5799]
## Context

Our link text CTAs use the single right-pointing angle quotation mark (›) at the end. Screen readers may read this out so [our CTAs](https://vanillaframework.io/docs/patterns/cta-block) sound like “ contact us single right-pointing angle quotation mark”. I suggest updating the CSS: 

.p-cta-link::after {
  content: ' ›';
  margin-left: 0.5rem;
}

Then this is properly coded as a styling element.

**Comment by dgtlntv** [comment]
Design systems contribution meeting:
- For this pattern we agree that the chevron should not be announced to screen readers. How it is implemented should be decided by the developer implementing it.
- For Pragma we should potentially consider to have a general approach towards announcing / not announcing icons and symbols more generally.
